src/test/java/sysc4806/group7/mini_shopify/MerchantControllerTest.java

```diff
@@ -18,6 +18,10 @@ public class MerchantControllerTest {
     @Autowired
     private MockMvc mockMvc;
 
+    @Autowired
+    MerchantRepository merchantRepository;
+
+
```

Comment:
why is this needed?

Reply:
it's not. I was testing something and forgot to remove it.